src/main/java/bit/javaoop/AbstractManager.java

```diff
@@ -0,0 +1,53 @@
+package bit.javaoop;
+import java.util.List;
+
+public abstract class AbstractManager extends AbstractEmployee {
+
+    private final Float budget = 100_000f;
+    private List <AbstractEmployee> EmployeeList = getEmployeeList();
+
+
+    public AbstractManager(String name, Float salary)
+    {
+       super(name,salary);
+    }
+
+
+    public abstract boolean canHire();
+
+    public boolean isSatisfied()
+    {
+        return getSalary()>10_000.0f;
```

Comment:
A proper formatting is important for the code to be readable. Here for example you lack spaces around `>`. If you use IntelliJ you can easily reformat the whole codebase by using a shortcut (cmd + alt + L on MacOS, something similar on Windows/Linux)